packages/editor/src/components/index.js

```diff
@@ -21,6 +21,7 @@ export { default as EntitiesSavedStates } from './entities-saved-states';
 export { useIsDirty as useEntitiesSavedStatesIsDirty } from './entities-saved-states/hooks/use-is-dirty';
 export { default as ErrorBoundary } from './error-boundary';
 export { default as LocalAutosaveMonitor } from './local-autosave-monitor';
+export { default as MainElementWarnings } from './main-element-warnings';
```

Comment:
Do we want to make this component a public API? Probably better to have it for internal use only for now.